### DISCUSSION TITLE
SimpleClans expansion removal

### DISCUSSION THREAD

**Discussion by RoinujNosde** [PlaceholderAPI/PlaceholderAPI#707] [(2) Support] [answered]
Hello,
Could you remove the [SimpleClans expansion](https://api.extendedclip.com/expansions/simpleclans/) from the eCloud?
It was made for an [old version](https://www.spigotmc.org/resources/simpleclans.5269/) of the plugin, which is no longer maintained.
[Mine](https://www.spigotmc.org/resources/simpleclans.71242/) is constantly getting updates and this expansion from eCloud causes issues with the one my plugin internally provides.

**Comment by Funny-cube** [comment, marked as answer]
Hi,

I've unverified the ecloud expansion which will prevent it showing ingame but still allow people to install it if they really want to manually.

Thanks,
Funnycube

**Reply by RoinujNosde** [reply]
Thanks
